### PR TITLE
add keras2StyleGuide

### PR DESCRIPTION
**Pull request by simanw304**
add doc for following layers:
Conv1D
Conv2D
Maximum
Minimum
AveragePooling1D
MaxPooling1D

### CODE REVIEW

File: docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+            (i.e. the number of output filters in the convolution).
+* `kernel_size`: An integer or tuple/list of a single integer,
+            specifying the length of the 1D convolution window.
+* `strides`: An integer or tuple/list of a single integer,
+            specifying the stride length of the convolution.
+* `padding`: One of `"valid"`, `"causal"` or `"same"` (case-insensitive).
+            `"valid"` means "no padding".
+            `"same"` results in padding the input such that
+            the output has the same length as the original input.
+            `"causal"` results in causal (dilated) convolutions, e.g. output[t]
```
Comment (hkvision) [inline docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md]: Do we support casual?

File: docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+            does not depend on input[t+1:]. Useful when modeling temporal data
+            where the model should not violate the temporal order.
+* `activation`: Activation function to use. See [here](activation/#available-activations) for available activation strings. If you don't specify anything, no activation is applied
+            (ie. "linear" activation: `a(x) = x`).
+* `use_bias`: Boolean, whether the layer uses a bias vector.
+* `kernel_initializer`: String representation of the initialization method for the weights of the layer. See [here](initialization/#available-initialization-methods) for available initialization strings. Default is 'glorot_uniform'.
+* `bias_initializer`: Initializer for the bias vector.
+* `kernel_regularizer`: An instance of [Regularizer](../../APIGuide/Regularizers/), (eg. L1 or L2 regularization), applied to the input weights matrices. Default is null.
```
Comment (hkvision) [inline docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md]: The link is 404... Using `[Regularizer](../../APIGuide/Regularizers/)` works in BigDL but not in Zoo...

File: docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+            `"same"` results in padding the input such that
+            the output has the same length as the original input.
+            `"causal"` results in causal (dilated) convolutions, e.g. output[t]
+            does not depend on input[t+1:]. Useful when modeling temporal data
+            where the model should not violate the temporal order.
+* `activation`: Activation function to use. See [here](activation/#available-activations) for available activation strings. If you don't specify anything, no activation is applied
+            (ie. "linear" activation: `a(x) = x`).
+* `use_bias`: Boolean, whether the layer uses a bias vector.
+* `kernel_initializer`: String representation of the initialization method for the weights of the layer. See [here](initialization/#available-initialization-methods) for available initialization strings. Default is 'glorot_uniform'.
+* `bias_initializer`: Initializer for the bias vector.
```
Comment (hkvision) [inline docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md]: String representation?

**Comment by simanw304** [comment]
The main structure refers to BigDL/Keras API doc. I modify the parameters according to keras2.

**Comment by hkvision** [comment]
Okay.

**Comment by simanw304** [comment]
http://10.239.47.211:18888/job/ZOO-PR-Validation/850/
jenkins passed

File: docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+            The ordering of the dimensions in the inputs.
+            `channels_last` corresponds to inputs with shape
+            `(batch, height, width, channels)` while `channels_first`
+            corresponds to inputs with shape
+            `(batch, channels, height, width)`.
+            It defaults to the `image_data_format` value found in your
+            Keras config file at `~/.keras/keras.json`.
+            If you never set it, then it will be "channels_last".            
+* `borderMode`: one of `"valid"` or `"same"` (case-insensitive).
+            Note that `"same"` is slightly inconsistent across backends with
```
Comment (zhichao-li) [inline docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md]: remove this line: "Note that...."

File: docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+            `strides` != 1.
+* `data_format`: Number of columns in the convolution kernel.
```
Comment (zhichao-li) [inline docs/docs/Keras2StyleAPIGuide/Layers/convolutional.md]: incorrect description?

**Comment by simanw304** [comment]
http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/856/
jenkins passed

**Pull request merged**